src/irmin/irmin.mli

```diff
@@ -1747,12 +1747,15 @@ module Private : sig
         ?node:(node -> unit Lwt.t) ->
         ?edge:(node -> node -> unit Lwt.t) ->
         ?skip:(node -> bool Lwt.t) ->
+        ?rev:bool ->
         unit ->
         unit Lwt.t
       (** [iter min max node edge skip ()] is the same as closure except that it
```

Comment:
Can you also clarify that comment -- `iter` is not exactly like `same as closure`. I think you mean the parameter of the function have the same meaning? I think we should state what this function is doing a bit more clearly. What about:

```
[iter ...] iterates over a subgraph in topological order. If [rev] is false (the default), we have the guarantee that all the predecessors of a node will be traversed before `node` is called.
```
We also need to say which guarantee do we have for `edge`: will it be called once both the source and the destination is traversed, or is it done lazily?

Reply:
I changed it, it might be too much text now. I would like to keep `closure graph` in the doc so as not too specify again the constraints in the doc of `closure` (too be specified [here](https://github.com/mirage/irmin/pull/908) )